README.md

```diff
@@ -32,6 +32,31 @@ Then you have to call the constructor with the correct dev/prod variables
 
 `var metrics = new Auth0Metrics('segmentKey', 'dwhEndpoint', 'website');`
 
+### Loader
+
+A script that will asynchronously load `auth0-metrics.js` is also provided.
+
+```html
+<script src="auth0-metrics-loader.js"></script>
+<script>
+  metricsLib.load({
+    segmentKey: 'segmentKey',
+    dwhEndpoint: 'dwhEndpoint',
+    label: 'website'
+  });
+</script>
+```
```

Comment:
I didn't add the example because I'm not sure about what is the motivation for the _sync_ case. What is the advantage of doing:

```html
<script src="auth0-metrics.js"></script>
<script src="auth0-metrics-loader.js"></script>
<script>
    metricsLib.load({
    segmentKey: 'segmentKey',
    dwhEndpoint: 'dwhEndpoint',
    label: 'website'
  });
</script>
```

Instead of:

```html
<script src="auth0-metrics.js"></script>
<script>
  var metricsLib = new Auth0Metrics('segmentKey', 'dwhEndpoint', 'website');
</script>
```